library/user/general/HackStats/payload.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
This payload works great. Unfortunately we can't accept it until the other two that are part of this pull request are put into their own directories. Hope to add it soon!